middleware_url_rewrite.go

```diff
@@ -113,34 +113,10 @@ func (u URLRewriter) Rewrite(thisMeta *tykcommon.URLRewriteMeta, path string, us
 			contextKey := strings.Replace(v[0], "$tyk_meta.", "", 1)
 			log.Debug("Replacing: ", v[0])
 
-			tempVal, ok := thisSessionState.MetaData.(map[string]interface{})[contextKey]
+			tempVal, ok := thisSessionState.MetaData[contextKey]
 			if ok {
-				var nVal string
```

Comment:
I looked into the same before - it's used in multiple places. This one was removed from master too, as there is no interface type here. The new code is:

```
val, ok := session.MetaData[contextKey]
if ok {
        newpath = strings.Replace(newpath, v[0],
                url.QueryEscape(valToStr(val)), -1)
}
```

Note that `valToStr` does nothing for strings, and the same applies to the deleted chunk of code.